lib/actions/ui.js

```diff
@@ -166,3 +171,32 @@ export function windowMove () {
     });
   };
 }
+
+export function openFile (path) {
+  return (dispatch, getState) => {
+    dispatch({
+      type: UI_OPEN_FILE,
+      effect () {
+        stat(path, (err, stats) => {
+          if (!err) {
+            // We need to use 'php-escape-shell' property this way
+            // until this eslint issue will be fixed:
+            // https://github.com/eslint/eslint/issues/6755
+            let command = shellEscape.php_escapeshellcmd(path);
+            if (stats.isDirectory()) {
+              command = 'cd ' + command + '\n';
```

Comment:
Right, since i still use es5 i'm not accustomed to use it, sorry. I'll change it tomorrow :)